test/zhr_devs/bakery_integration/command_runner_test.exs

```diff
@@ -0,0 +1,93 @@
+defmodule ZhrDevs.BakeryIntegration.CommandRunnerTest do
```

Comment:
I know we shouldn't really test for it, but do we kill port after 60 seconds at each run?

Reply:
But why we are limiting to the 60 seconds? We are gauranteed to receive the `:DOWN` message once `port` will exit? Why bother with strict timeouts?